core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReaderIterator.java

```diff
@@ -533,15 +535,19 @@ static void trackScanning(Map<KeyExtent,List<Range>> failures,
       Map<KeyExtent,List<Range>> unscanned, MultiScanResult scanResult) {
 
     // translate returned failures, remove them from unscanned, and add them to failures
-    Map<KeyExtent,List<Range>> retFailures = Translator.translate(scanResult.failures,
-        Translators.TKET, new Translator.ListTranslator<>(Translators.TRT));
+         Map<KeyExtent, List<Range>> retFailures = scanResult.failures.entrySet().stream()
```

Comment:
We've already discussed this in person, but I'm including a comment here so anyone else can chime in; these kinds of transformation operations are good candidates for being moved to a separate utility class that supports multiple types of collection transformations using lambdas and function parameters. This would allow shorter function calls to improve readability, as well as make it available to the project. I'd be happy to help out with this effort.